natgeo/mortar
# [SYTO-729] Build video playlist component in Web Tools/Mortar (Elevate AEM's component)

https://jira.natgeo.com/browse/SYTO-729

#### How to test
1. ``npm i``
2. ``gulp serve``
3. Go to http://localhost:3000/components/
4. Check the video playlist's functionality 
5. :elephant:
